tensorflow/compiler/jit/xla_gpu_device.cc

```diff
@@ -103,7 +106,8 @@ Status XlaGpuDeviceFactory::CreateDevices(
   (void)registrations;
 
   // XXX FIXME devise a way to cope with multiple platforms
```

Comment:
remove "XXX FIXME"?

Reply:
will do.

either in this PR (if CI fails and I need to push out another commit) or in the next PR I file